core/src/main/java/com/novoda/accessibility/ActionsAlertDialogCreator.java

```diff
@@ -14,29 +16,37 @@
 
     private final Actions actions;
 
-    public ActionsAlertDialogCreator(Context context, int title, Actions actions) {
+    public ActionsAlertDialogCreator(Context context, Actions actions) {
+        this(context, NO_TITLE, actions);
+    }
+
+    public ActionsAlertDialogCreator(Context context, @StringRes int title, Actions actions) {
```

Comment:
if any project needs it, it can be added :grin:

For now, the ones that use it will be static strings in resources.